cts/cts-cli.in

```diff
@@ -3381,7 +3381,11 @@ function print_or_remove_file() {
   if [[ ! $(diff -wq $test_home/cli/regression.$1.exp "$TMPFILE") ]]; then
     rm -f "$TMPFILE"
   else
-    echo "    $TMPFILE"
+    if [ $verbose -eq 0 ]; then
+      echo "    $TMPFILE"
+    else
+      cat "$TMPFILE"
+    fi
```

Comment:
In case there's more than one file to output, let's add a couple blank lines and the file name as a header